src/setupTests.js

```diff
@@ -1,10 +1,11 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import { configure, mount, shallow } from 'enzyme';
+import { cleanup } from 'react-testing-library';
```

Comment:
Because JavaScript. 
Look at the line above. It's called `configure` :) 

I don't like `import * as ... from '...'` because when tree shaking and partial imports can be used that's an anti-pattern. 

Also, I've seen a lot of code where they accept this mess and leave it as `import { commonword } from 'javascript-lacks-namespaces'`

Reply:
Incidentally and whimsically, how about [`is`](https://github.com/mozilla/delivery-console/blob/master/src/workflows/recipes/components/RecipeForm.js#L3) :)

Reply:
Consistency is good, but repeating mistakes for the sake of consistency isn't useful. We can improve over time. Just because there are bad examples already in the code doesn't mean we can't do better going forward.